packages/design-system-react-native/src/components/BadgeWrapper/README.md

```diff
@@ -0,0 +1,187 @@
+| ----------------- | -------- | ------------------------ |
+| `React.ReactNode` | Yes      | Badge element to render. |
+
+---
+
+### `position`
+
+Optional preset to determine badge placement.
+
+| TYPE                   | REQUIRED | DEFAULT                            |
+| ---------------------- | -------- | ---------------------------------- |
+| `BadgeWrapperPosition` | No       | `BadgeWrapperPosition.BottomRight` |
+
+Options:
+
+- `TopRight`
+- `BottomRight`
+- `BottomLeft`
+- `TopLeft`
+
+---
+
+### `positionAnchorShape`
+
+Specifies the shape of the anchor to adjust badge alignment.
+
+| TYPE                              | REQUIRED | DEFAULT                                    |
+| --------------------------------- | -------- | ------------------------------------------ |
+| `BadgeWrapperPositionAnchorShape` | No       | `BadgeWrapperPositionAnchorShape.Circular` |
+
+Options:
+
+- `Circular`
+- `Rectangular`
+
+---
+
+### `positionXOffset`
+
+Horizontal adjustment for the badge position.
+
+| TYPE     | REQUIRED | DEFAULT |
+| -------- | -------- | ------- |
+| `number` | No       | `0`     |
+
+---
+
+### `positionYOffset`
+
+Vertical adjustment for the badge position.
+
+| TYPE     | REQUIRED | DEFAULT |
+| -------- | -------- | ------- |
+| `number` | No       | `0`     |
+
+---
+
+### `customPosition`
+
+Allows complete control over badge positioning using style values.
+
+| TYPE                         | REQUIRED | DEFAULT     |
+| ---------------------------- | -------- | ----------- |
+| `BadgeWrapperCustomPosition` | No       | `undefined` |
+
+Example:
+
+```ts
+{
+  top: 5,
+  right: 10,
+}
+```
+
+---
+
+### `twClassName`
+
+Tailwind class names to apply to the outermost wrapper.
+
+| TYPE     | REQUIRED | DEFAULT |
+| -------- | -------- | ------- |
+| `string` | No       | `''`    |
+
+---
+
+### `style`
+
+Optional style override for the outermost wrapper.
+
+| TYPE                   | REQUIRED | DEFAULT |
+| ---------------------- | -------- | ------- |
+| `StyleProp<ViewStyle>` | No       | `null`  |
+
+---
+
+## Usage
+
+### Basic Usage
+
+```tsx
+<BadgeWrapper badge={<BadgeStatus status={BadgeStatusStatus.Active} />}>
+  <AvatarAccount address="0x..." />
+</BadgeWrapper>
+```
+
+---
+
+### Preset Positions
+
+```tsx
+<BadgeWrapper
+  position={BadgeWrapperPosition.TopRight}
+  badge={<BadgeCount count={8} />}
+>
+  <AvatarAccount address="0x..." />
+</BadgeWrapper>
+```
+
+---
+
+### Custom Position
+
+```tsx
+<BadgeWrapper
+  customPosition={{ top: 4, right: 10 }}
+  badge={<BadgeIcon variant={BadgeIconVariant.Send} />}
+>
+  <AvatarNetwork name="ETH" src={{ uri: 'https://...' }} />
+</BadgeWrapper>
+```
+
+---
+
+### Anchor Shape Adjustment
+
+```tsx
+<BadgeWrapper
+  positionAnchorShape={BadgeWrapperPositionAnchorShape.Rectangular}
+  badge={<BadgeNetwork name="ETH" src={{ uri: 'https://...' }} />}
+>
+  <AvatarNetwork name="ETH" src={{ uri: 'https://...' }} />
+</BadgeWrapper>
+```
```

Comment:
I noticed we don't have any docs covering Xoffset and Yoffset usage?

Reply:
![Screenshot 2025-03-27 at 9 07 14 AM](https://github.com/user-attachments/assets/9a1b4f09-eb40-43ec-83a5-ca7b34d34fa2)

Reply:
Right, but no code examples on how to use these props?